packages/prisma-client/src/types.ts

```diff
@@ -1,9 +1,10 @@
-import type { Asset as DbAsset } from "@prisma/client";
+import type { Asset as DbAsset, Project as BaseProject } from "@prisma/client";
 export { Location } from "@prisma/client";
-export type { InstanceProps, Project, User, Breakpoints } from "@prisma/client";
+export type { InstanceProps, User, Breakpoints } from "@prisma/client";
 
 export type Asset = DbAsset & {
-  path: string;
+  path?: string;
   status?: "uploading" | "uploaded";
 };
+export type Project = BaseProject & { assets?: Asset[] };
```

Comment:
Isn't prisma supposed to know that assets are part of the Project type and give it to us automatically?